service.yml

```diff
@@ -0,0 +1,88 @@
+name: sidekiq-publisher
```

Comment:
I recommend dropping this file as we don't use our internal build pipeline for open source projects.